src/deadline/job_attachments/incremental_downloads/asset_downloader_with_manifest_aggregation.py

```diff
@@ -0,0 +1,144 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
It would be good to rename the `incremental_downloads` directory to `_incremental_downloads` and keep these functions internal-only while we're working out details.